docker/Dockerfile.browse

```diff
@@ -0,0 +1,19 @@
+FROM node:4.4.4
+LABEL node-version="4.4.4"
+
+ARG PLUGINS_REPO
+EXPOSE 4200
+WORKDIR /usr/src
+
+# Update npm
+RUN npm install -g npm@3.9.0
+
+# Clone repos
+RUN git clone https://github.com/dcos/dcos-ui.git /usr/src/dcos-ui
```

Comment:
Why are we downloading the complete repository instead of copying it into the container?